src/Configuration/Config.Azure.KeyVault.Secrets/src/Microsoft.Extensions.Configuration.Azure.KeyVault.Secrets.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Azure KeyVault configuration provider implementation for Microsoft.Extensions.Configuration.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>$(PackageTags);azure;keyvault</PackageTags>
+    <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+    <HasReferenceAssembly>false</HasReferenceAssembly>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <!-- Remove the properties below after releasing 3.1.0 of the package -->
+    <PreReleaseVersionLabel>$(AzureKeyVaultSecretsPreReleaseVersionLabel)</PreReleaseVersionLabel>
+    <DotNetFinalVersionKind />
+    <PatchVersion>0</PatchVersion>
```

Comment:
```suggestion
```

We've decided to leave the version alone, so it'll have a patch version for the first release.